src/libs/sfdk/buildengine.h

```diff
@@ -87,6 +87,8 @@ class SFDK_EXPORT BuildEngine : public QObject
     void setSharedSrcPath(const Utils::FileName &sharedSrcPath, const QObject *context,
             const Functor<bool> &functor);
 
+    QString sshHost() const;
```

Comment:
hm, there is still a big question mark hanging above the properties that primarily exists on VirtualMachine side but are exposed either on BuildEngine or both VirtualMachine and BuildEngine. There is e.g. "FIXME internal" on VirtualMachine::sshParameters suggesting that exposing it on BuildEngine would be more correct but this is still to be investigated.

So for now I would prefer keeping the existing API as is and use `engine->virtualMachine()->sshParameters().host()` where we need the host.

Reply:
I somewhat disagree, but fine, I'll change it

Reply:
And to be clear about what I disagree with you about: I think we should either expose port and host or expose neither. And I think the best option would be what you already suggested: expose sshParameters.

Reply:
Yup, it's just a bit disrupted by the fact that there is (still) the wwwPort too, so I am not totally sure yet the final state will be with sshParameters exposed here.